### DISCUSSION TITLE
Jamulus crashes on macOS

### DISCUSSION THREAD

**Discussion by harmonk** [jamulussoftware/jamulus#3452] [Technical Support & Help]
Jamulus [3.11] has been crashing for about a week now. I open it, it runs for seconds, then blink, gone.
I am on a Mac Macbook Pro, Sequoia 15.2 - updated recently, could be related but find no other reports of problems. And i believe Jamulus ran fine for a short time before this problem began.
I have reinstalled Jamulus several times, deleting all related files each time.  Restarted the computer. 
I use J with Zoom often, thought that might be an issue but J crashes with no other apps open.
Anybody else? Any ideas?

**Comment by chrisrimple** [comment]
Jamulus requires that the device in use support 48KHz bitrate.  Try this:

1. Run Applications > Utilities > Audio MIDI Setup
2. If the Audio Devices window is not visible, show it with Window > Show MIDI Devices
3. On the left side, select each device and confirm that both Input and Output format = 48000 Hz

#3 is particularly important for your default Input and Output devices (usually Mac Internal Microphone and Mac External Speakers, but can vary by device).

Then launch Jamulus.  I'm just guessing, but maybe it's defaulting to a device that's not set to 48KHz and that's causing a crash.

**Reply by harmonk** [reply]
Thanks for the idea. I would metion first that i have changed nothing lately, and Jamulus worked fine until very recently.
I am guessing you mean AUDIO devices? Not MIDI? I checked the builtin speakers and microphone. These are indeed 48.000. You might mean MIDI devices, but Jamulus crashes even if no external device is plugged in. AND if they ARE plugged in, doesn't seem to make a difference. Normally what i use is a Focusrite interface in/out using USB, already for years. Again, nothing has changed lately.

Starting up Jamulus now, new weird behavior: the Connect and Settings windows are open, and trying to click them closed leaves blank windows, shadows kind of, that i can't get rid of.

Anyway, no change in the crashing.

**Reply by chrisrimple** [reply]
Yes, I was referring to audio devices.  Apple's Audio MIDI Setup app handles both.

After starting Jamulus, do you see it running in the Apple > Force Quit list of applications that are running?  Do you still see it in that list after closing the Connect and Settings windows?

**Reply by ann0see** [reply]
> the Connect and Settings windows are open, and trying to click them closed leaves blank windows, shadows kind of, that i can't get rid of.

This seems like an OS issue. Is your graphics card ok?
Otherwise, does the legacy version (https://github.com/jamulussoftware/jamulus/releases/download/r3_11_0/jamulus_3.11.0_mac_legacy.dmg) of Jamulus also lead to errors?

**Reply by harmonk** [reply]
https://github.com/user-attachments/assets/0c1e7e10-a2f6-4596-903e-e4a709992905

A quick vid of how the windows look.

**Reply by ann0see** [reply]
Hmm. This looks like the window doesn't get closed correctly by Qt.

**Reply by harmonk** [reply]
My graphics seem to be fine. No other issues, things look fine, we watch videos and play games and so on.
I downloaded that legacy version [though doesn't it have the same version number as the latest Jamulus version?] and installed. Same reult. It ran for about two minutes before it crashed the first time, then crashed immediately from then on.

**Reply by harmonk** [reply]
With this legacy version, at least i don't have the shadow window issue anymore.

**Reply by ann0see** [reply]
> With this legacy version, at least i don't have the shadow window issue anymore

Which means that it could be a Qt related bug. 

I don't have an (apple silicon) Mac on 15.2 unfortunately. Currently, it seems as if it only occurs on your hardware - which I find odd. 
@chrisrimple You can't reproduce this problem either?

**Comment by ann0see** [comment]
As soon as Jamulus crashes, does an error message from macOS show up? It would be good to see the crash logs. 

Otherwise, open macOS console and look for any errors from Jamulus reported there.

**Reply by harmonk** [reply]
I must admit i am not sure where to find crash logs. I have made at two reports to Apple about this, and so i look in Console and see on the left 'Crash Reports', but there is nothing relevant there. Where must i look?

**Reply by harmonk** [reply]
I searched in Console for Jamulus, nothing found.

**Reply by ann0see** [reply]
Did you use https://support.apple.com/en-us/guide/console/welcome/mac

I would leave it open and view it in real time until a crash happens again.

**Comment by ann0see** [comment]
Some questions:
* Which audio hardware did you try? Internal devices + Focusrite?
* What is the exact model + year of your Mac? Is it Intel or Apple silicon?
* Did you remove zoom already and tried if there's a crash? Zoom might run in the background. Are there any other apps which access your audio gear?
* Does it only occur with the latest Jamulus version?
* Did you try opening Jamulus via the terminal already? Do you see any crash in the terminal?

**Reply by harmonk** [reply]
Audio hardware: with and without Focusrite. It doesn't seem to matter what hardware is present, or not. Mainly i have been testing with nothing attached.
Macbook Pro 14in 2021, Apple M1 Max chip, Sequoia 15.2.
Activity Monitor says that Zoom is not running at all. I will try removing it completely and report back. I used Jamkazam before with the same setup, but that was years ago. Logic Pro too, sometimes i hook it up with the FocusRite. But i have been doing these things for years without an issue.
I posted about about downloading a legacy version [(https://github.com/jamulussoftware/jamulus/releases/download/r3_11_0/jamulus_3.11.0_mac_legacy.dmg) ] and tried it. Same result.
Opening Jamulus via Terminal? Tried it just now, opening that legacy version. Jamulus has stubbornly refused to crash ;) monitoring now.

**Comment by harmonk** [comment]
And it crashed again, after being on for maybe 30 minutes. With Zoom also on.
Weird coincidence: It crashed twice finally. Both times, i opened Signal and sent a message. Boom, Jamulus crashed. Twice.
This, for all openness. Who knows, could be relevant.

**Reply by ann0see** [reply]
Did you rule out a hardware issue: https://support.apple.com/en-us/102550?

**Reply by harmonk** [reply]
Have just done that, and no issues were reported. Code ADP000.

**Comment by chrisrimple** [comment]
I'm not experiencing similar problems with 3.11 on MacOS Sequoia 15.1.1, but I have not yet upgraded to 15.2, so can't say yet if that's the issue.  2021 MacBook Pro M1 Max.

**Comment by ann0see** [comment]
Another suggestion: does the Mac artifact from https://github.com/jamulussoftware/jamulus/actions/runs/12525153233 work? That’s built with the latest Qt version.

**Reply by harmonk** [reply]
Same thing happens.

**Reply by ann0see** [reply]
Ok. So I think we'd need someone else to test on apple silicon latest macOS version.

Unfortunately I don't own an apple silicon device. This would allow us to find out if it's specific to only your device. 

Can you try installing macOS on an external SSD and check if the issue persists? 

It might be dangerous - so backup first.

**Reply by chrisrimple** [reply]
It appears that MacOS 15.2 is available to me now, so I'll upgrade and test to see if I can duplicate.

**Reply by harmonk** [reply]
Thanks. I have a bootable backup on an external HD. Would that work for this?

**Reply by ann0see** [reply]
Yes. I'd boot from this and then check if Jamulus also crashes there. In the best case, you'd have a clean install of macOS somewhere and then check if Jamulus still crashes.

**Reply by harmonk** [reply]
I shall try this today.
But, i just made a new user account and logged in and started Jamulus. It crashed within a minute.

**Comment by harmonk** [comment]
Ok, finally got a log. I hope. Jamulus kept NOT crashing when i could test it. It always crashes in the end, but sometimes it would be more than 30 minutes. Finally turned on log recording, and J crashed. So i have a log.
How do i post it? In what form? And it's long, even though i filtered for 'Jamulus'. Please let me know and i will post it.

**Reply by ann0see** [reply]
You can check the log for sensitive info, remove it and then upload it as GitHub gist https://gist.github.com/ or past the content as comment here or upload the file to a cloud service.

**Reply by harmonk** [reply]
The log is here: https://gist.github.com/harmonk/e37e2ae86711fdf0b81c51fec59db12f
Hope it's useful, please let me know.

**Reply by ann0see** [reply]
> https://gist.github.com/harmonk/e37e2ae86711fdf0b81c51fec59db12f#file-gistfile1-txt-L72

Did you open the legacy version first? This line says the binary is not signed correctly. Probably not the problem.

**Reply by ann0see** [reply]
> Received state update for 5655 (app<application.app.jamulussoftware.Jamulus.79657781.79657889(502)>, none-NotVisible
default	19:44:02.473407+0100	powerd	Process Jamulus.5655 ClientDied PreventUserIdleDisplaySleep "Jamulus provides low latency audio processing and should not be inturrupted by system throttling." age:00:00:07  id:21474876446 [System: PrevIdle PrevDisp DeclUser IntPrevDisp kDisp]
default	19:44:02.477853+0100	runningboardd	Invalidating assertion 190-130-39769 (target:[app<application.app.jamulussoftware.Jamulus.79657781.79657889(502)>:5655]) from originator [osservice<com.apple.powerd>:130]
default	19:44:02.489043+0100	runningboardd	[app<application.app.jamulussoftware.Jamulus.79657781.79657889(502)>:5655] termination reported by launchd (2, 13, 13)
default	19:44:02.489442+0100	runningboardd	Removing process: [app<application.app.jamulussoftware.Jamulus.79657781.79657889(502)>:5655]
default	19:44:02.490171+0100	runningboardd	Removing launch job for: [app<application.app.jamulussoftware.Jamulus.79657781.79657889(502)>:5655]
default	19:44:02.490776+0100	runningboardd	Removed job for [app<application.app.jamulussoftware.Jamulus.79657781.79657889(502)>:5655]
default	19:44:02.490813+0100	runningboardd	Removing assertions for terminated process: [app<application.app.jamulussoftware.Jamulus.79657781.79657889(502)>:5655]

Did it crash at this time?

It could be some kind of changed system behavior where macOS doesn't correctly "know" that it shouldn't put the app to sleep.

Does the crash always happen if the app is not in focus?

https://github.com/jamulussoftware/jamulus/issues/23

**Reply by harmonk** [reply]
I can't say exactly but roughly around this time, yes.
Sleeping would be odd, because i stayed actively using the computer and Jamulus.
J crashes at any time, in focus or not.

**Reply by harmonk** [reply]
> > https://gist.github.com/harmonk/e37e2ae86711fdf0b81c51fec59db12f#file-gistfile1-txt-L72
> 
> Did you open the legacy version first? This line says the binary is not signed correctly. Probably not the problem.

That's a good question. I have switched back and forth among different versions. If i look in this app, in the info.plist i see 'CFBundleVersion 3.11.0dev-eb09a25'. Is that a help?

**Reply by ann0see** [reply]
> Sleeping would be odd, because i stayed actively using the computer and Jamulus.
J crashes at any time, in focus or not.

Still, macOS might try to put the app into some kind of napping mode (App Nap, https://developer.apple.com/library/archive/documentation/Performance/Conceptual/power_efficiency_guidelines_osx/AppNap.html) to not drain too much battery/power. iOS is a lot more aggressive but macOS has this feature too.

**Reply by ann0see** [reply]
> If i look in this app, in the info.plist i see 'CFBundleVersion 3.11.0dev-eb09a25'. Is that a help?

Unfortunately not. If it's legacy, it should only support Intel Macs, else it should be Native/Apple architecture/... in activity monitor.

**Comment by ann0see** [comment]
@AndersGoran
First of all sorry for the ping. I hope it's ok to ping you. 

Since you have more in depth macOS knowledge - I assume - do you have any idea what could lead to a macOS crash?

It doesn't seem to be audio hardware related. MacOS 15.2 seems to be the culprit.

It feels a bit like an issue between some interfering software or setting. 
Maybe you have an idea?

**Reply by AndersGoran** [reply]
No problem pinging me, but I have to check my notification settings because I didn't see this until I happened to open the GitHub app. 
I've seen the thread and I was thinking I should update my "spare" Mac to 15.2 and try to reproduce this. It's a fairly clean system that I can muck around with. Just have to get around to it...

**Comment by chrisrimple** [comment]
I've updated to MacOS Sequoia 15.2 and have run Jamulus for 30+ minutes (connected to a server) without a crash.  Not a definite test showing that it's a problem unique to @harmonk's computer, but perhaps indicative.

**Reply by harmonk** [reply]
Do you use Zoom? Perhaps start that up at the same time? It's unlikely, but there is a faint possibility there is a connection.
One thing more for me, starting from the external drive. Today.
Otherwise i am out of ideas. Restarted many times. I have deleted Jamulus, reinstalled, tried different older versions, which used to work. It doesn't seem hardware related, because this happens no matter what is connected or not, and i did an Apple hardware test and everything seemed fine. It actually doesn't seem like like a conflict with other software, because the crash happens sometimes immediately and sometimes after a half hour or more, and wouldn't it happen always at more or less the same time if it were software related? I am not an expert.... 
My best guess if that it is the OS. But then many other people would be be having the same problem.
Security software? I have Little Snitch running always, but i have allowed all connections for Jam. Browsers have Ublock and Noscript and Ghostery. I also have the apps Adblock and Adblock Pro. ClamxAv. Techtool, Tinkertool. These are the ones that are maybe a bit special that i would consider possibilities. All my other apps are totally normal, OS stuff and browsers and music software. 
???

**Reply by ann0see** [reply]
> Techtool, Tinkertool.

While I don't know these, but maybe they changed something in the system making Jamulus crash. 

I think in this case it would really make sense to boot from an external SSD from a clean macOS instead of a backup.

**Reply by harmonk** [reply]
That does make sense. Ok, have to make one and then test. Will get back to you.

**Comment by ann0see** [comment]
Something else. We had a very odd issue 4 years ago: https://github.com/orgs/jamulussoftware/discussions/1447

Another question: do you use multiple monitors? Specifically: do you use display link?

**Reply by harmonk** [reply]
Well, that odd issue seemed to be a physical thing where his cymbal hit a cable or something.....
I do not use multiple monitors. The only thing -- i hook up to a video monitor to watch a film sometimes. But that external monitor was never connected when Jam crashed.

**Reply by ann0see** [reply]
> Well, that odd issue seemed to be a physical thing where his cymbal hit a cable or something.....

Yes exactly. What I wanted to convey with this is that the issue could lie somewhere unexpected. 

> I do not use multiple monitors. The only thing -- i hook up to a video monitor to watch a film sometimes. 

Ok. But then that problem is probably out of scope. 

I know that DisplayLink sometimes introduces weird behavior. It hid my password manager on an external screen for security reasons.

**Reply by harmonk** [reply]
"Yes exactly. What I wanted to convey with this is that the issue could lie somewhere unexpected."
;) Definitely. But i don't know what it could be. No external things are connected. I did test with that Focusrite connected, but the behavior was the same.

**Reply by harmonk** [reply]
Right now i am testing with version 3.10.0. It's been on for a while, interacting with it often, Zoom also running. Still there....

**Reply by harmonk** [reply]
It crashed. I reinstalled the OS. Tested that same older version, it crashed.
Now i downloaded the latest version 3.11.0 and testing now.

**Reply by harmonk** [reply]
btw - might this be relevant? Little Snitch tells me that the app is not in the normal Applications folder.
![jamulus LS 1](https://github.com/user-attachments/assets/9ee91bc2-2c92-4c33-8f11-208a6ec04049)
![jamulus LS 2](https://github.com/user-attachments/assets/801cb2e5-6519-4880-804d-ba6ebe2ca6c8)

**Reply by harmonk** [reply]
For me Jamulus appears in the Applications folder, as normal.

**Reply by harmonk** [reply]
....aaaaand it crashed.

**Reply by ann0see** [reply]
> Little Snitch tells me that the app is not in the normal Applications folder.

Please copy the Jamulus.app file to Applications again. Maybe you had an old version there.

**Reply by ann0see** [reply]
> I reinstalled the OS. 

Is it a clean install or not?

**Reply by ann0see** [reply]
Could you please also take a screenshot of the error which opens if the app crashes. If there is no error message saying something like "Jamulus quit unexpectedly" we're dealing with something else than a crash.

**Reply by harmonk** [reply]
Well, i delete Jamulus [many times] from the Applications folder. Reinstall it, in the usual way. It appears in the Appl folder. But Little Snitch claims it's in that other place. But there it is in the Appl folder.
I followed that path from LS. Nothing there. Until i started Jam up, then a disk appeared in that folder. I see Jamulus there now. Version 3.11, just like the one in the Appl folder. So i think they are the same.

**Reply by harmonk** [reply]
> > I reinstalled the OS.
> 
> Is it a clean install or not?

Um, not sure what that is, sorry. I reinstalled it using the Mac Recovery mode, the same OS.

**Reply by harmonk** [reply]
> Could you please also take a screenshot of the error which opens if the app crashes. If there is no error message saying something like "Jamulus quit unexpectedly" we're dealing with something else than a crash.

This shows only sometimes. Lately not at all. If i can catch it i will post.

**Reply by AndersGoran** [reply]
That thing about the app running in an unexpected folder is apparently a rather old security feature of 10.12 and onwards, where it runs a newly downloaded app in a random location. (https://forums.developer.apple.com/forums/thread/724969)

I've just managed to update my Mac mini to 15.2; it never had Jamulus before and is now running the official release  for the first time, indeed from a random path under `/private/var`.

I was going to suggest opening the app from a shell script in Terminal - if there is something printed to stdout/stderr when it crashes, it might be easier to catch that in the Terminal. 

Open Terminal and create a script with a suitable name, e.g. `launch_jamulus.sh` with the following content - one single line: 
```
exec /Applications/Jamulus.app/Contents/MacOS/Jamulus "$@"
```
and then launch it from the same Terminal window:
```
chmod +x launch_jamulus.sh
./launch_jamulus.sh
```
Fingers crossed that this will work with your shell, which I think is `zsh` by default. You may have to adjust it slightly.

**Reply by ann0see** [reply]
> Um, not sure what that is, sorry. I reinstalled it using the Mac Recovery mode, the same OS.

Ok. That's just a soft re install of the OS. 
This means that the OS files should be clean now. However, this does not reset settings to factory defaults.

I would recommend to install macOS without any data on another drive so the whole OS + data on your main drive is not touched at all. The new disc is clean and has no data: https://www.macworld.com/article/672585/how-to-install-macos-on-an-external-drive.html

This process is dangerous if you wrongly reformat the internal drive instead of an external one.

**Reply by AndersGoran** [reply]
15.2 on its own is probably _not_ enough to cause a crash, at least not for me. I've had the 3.11.0 client running since last night - connected to a server, and this morning I started playing audio to the server as well. Still fine after 10 hours.

**Reply by ann0see** [reply]
@AndersGoran thank you for testing out. 

@harmonk do you have an alternative computer where you could use Jamulus on? Otherwise, you'd probably either need to remove or reset the apps in question until you find the culprit or install apps one after each other on a clean install.

**Reply by harmonk** [reply]
I will test this today with a newly installed clean external boot drive. Sorry that it goes so slowly, and i appreciate the help and attention here.

**Reply by harmonk** [reply]
Just want to report. I am having difficulty making a bootable external drive. Don't know why. I follow instructions from here and Apple, seems straightforward, but isn't apparently. I will keep trying.

**Reply by harmonk** [reply]
> That thing about the app running in an unexpected folder is apparently a rather old security feature of 10.12 and onwards, where it runs a newly downloaded app in a random location. (https://forums.developer.apple.com/forums/thread/724969)
> 
> I've just managed to update my Mac mini to 15.2; it never had Jamulus before and is now running the official release for the first time, indeed from a random path under `/private/var`.
> 
> I was going to suggest opening the app from a shell script in Terminal - if there is something printed to stdout/stderr when it crashes, it might be easier to catch that in the Terminal.
> 
> Open Terminal and create a script with a suitable name, e.g. `launch_jamulus.sh` with the following content - one single line:
> 
> ```
> exec /Applications/Jamulus.app/Contents/MacOS/Jamulus "$@"
> ```
> 
> and then launch it from the same Terminal window:
> 
> ```
> chmod +x launch_jamulus.sh
> ./launch_jamulus.sh
> ```
> 
> Fingers crossed that this will work with your shell, which I think is `zsh` by default. You may have to adjust it slightly.



I did this, indeed zsh. Jamulus started up fine, is still sitting there now, will report what happens. I expect a crash. So, how do i get at that stdout/stderr report?

**Reply by AndersGoran** [reply]
If it crashes, and it prints anything, it'll appear right there in the Terminal window. You can just copy & paste it.

**Reply by ann0see** [reply]
> I am having difficulty making a bootable external drive

Did you format your external drive as APFS?

**Reply by bigzedd** [reply]
I'm having the same issue with Jamulus Client app crashing. The Server app starts with no issues but the Client app just "flashes" the Jamulus app windows and nothing. I tried running with Rosetta and the same result. I'm running with an Apple Macbook Pro M2 Pro chip and Sequoia 15.2. Jamulus version 3.11.0

**Reply by ann0see** [reply]
@bigzedd Could you please post a list of apps you installed?

**Reply by bigzedd** [reply]
@ann0see For Music I've installed Ableton Live 11 Intro, Amplitube 5, Open Broadcaster Software (OBS). I've also installed VSTs and a plugin for virtual audio out Blackhole 2ch (48.0 kHz). For web development I've installed Node.js, VSCode, iTerm2, GIMP for image editing and a couple database desktop clients. I have Zoom, Firefox, Chrome installed as well.

**Reply by ann0see** [reply]
So the only similarity between your two installs is Zoom?

**Reply by ann0see** [reply]
What if you either completely remove Zoom or select totally different audio devices in both apps?

**Reply by AndersGoran** [reply]
And I _don't_ have Zoom on my 15.2 system. I should install that, then... 

**Update:** the currently available version of "Zoom Workplace" freshly installed seems to make no difference in my case. Jamulus is still fine here, even with Zoom running, with the same audio interface.

**Reply by bigzedd** [reply]
@ann0see Update I'm able to connect to public servers but the app crashes when I try to connect to a local server. Running server with Jamulus Server app or running server via command line `/Applications/Jamulus.app/Contents/MacOS/Jamulus --server` FYI In the past I ran my server locally and was able to jam with a few select friends using port forwarding. But it seems like I'm no longer able to do that with this new macbook pro.

**Reply by ann0see** [reply]
Start the client via CLI too. If it crashes, hopefully we get some error message.

**Reply by harmonk** [reply]
Of all those apps bigzedd mentions, i have only GIMP installed, along with Zoom of course. 
Not such a programmer. I use Logic. Have also JamKazam and Koord's app. OBS, and also Lightworks, BlackMagic and DaVinci Resolve but almost never used them. Finale and Dorico with accompanying Steinberg stuff. Firefox, Safari used often. Chrome, Opera installed too. QBitTorrent. Little Snitch. Not using a VPN, even though i should.

**Reply by harmonk** [reply]
I cannot yet get an external bootable drive to work. I have tried different drives, and each time the same. The installation process seems to work and yet when i try to select that disk to boot from, i click 'Continue' under that disk icon, and nothing happens. The 'Continue' button relights and no further action. Only if i click the button under my internal drive does it continue with startup.
only thing i might mention: After the installation on the external drive, it finishes and i see a window with 'Restart' and no other options. I am terrified that this will then [re]install the OS on my INTERNAL drive, erasing everything. So i don't do that. Should i? I can't find any reference to this on any site, about ten diffent ones so far.

**Reply by ann0see** [reply]
Unfortunately, I'm not sure how modern Macs handle this. 
If you prefer a video: https://youtube.com/watch?v=ZLqpjO_77V4

Might help. It does not use recovery mode.

**Reply by bigzedd** [reply]
> Start the client via CLI too. If it crashes, hopefully we get some error message.

@ann0see Tried both ways with client and without `/Applications/Jamulus.app/Contents/MacOS/Jamulus`

**Reply by ann0see** [reply]
Does it print an error message in terminal?

**Reply by bigzedd** [reply]
@ann0see It doesn't crash which is a good sign. However, it say's "TRYING TO CONNECT" and just hangs. I tried using my local ip and port  as well as the domain I used with port forwarding. There's no error logged in the terminals. Just to clarify this is only happening when I try to connect to a local Jamulus server. Thank you for you help folks!

**Reply by ann0see** [reply]
> However, it say's "TRYING TO CONNECT" and just hangs.

This hints at a sound card issue if there's no network problem. 

Are there any other sound devices which are in use?

**Reply by bigzedd** [reply]
@ann0see Okay I deleted all input output sources except my interface and default sound devices and restarted my laptop.
1. I only start the Jamulus Client
2. I connect to a public server.  I can hear folks and they can hear me
3. I disconnect from that public server
4. I then try to re-connect again. However, instantly when I click the "Connect" button, the app crashes :-( and crashes every time after that when I click "Connect". I'm not trying to connect to my local server just public servers.
5. Also, after waiting for some time for the connection dialog to load servers, the app crashes
https://github.com/user-attachments/assets/315f45a7-c1ac-49d8-82f8-fb6d0ca6a72a

**Reply by ann0see** [reply]
Thanks for the video. 
At least for you it seems to be reproducible. 
Cold you please start the client via terminal and show the terminal output alongside the app in a video like this?

**Reply by bigzedd** [reply]
@ann0see I have some good news to share. I'm able to connect to public and my local server without any errors. I can use the client/server apps or use the terminal command line options.

**TL;DR**
The crashes were happening because innitially I tried connecting to my local server which errored and caused a "cached" response every time after. I say "cached" but I suspect it's just that the server kept running in the system memory.

To get it to run with my local server, I first have to connect using 127.0.0.1:22124. After successfully connecting, I can disconnect and then connect via my public ip or vanity domain with port forwarding. I was able to jam again with my mates tonight with out any issues.

Thank you for all your help and patience.

Respectfully.
-Z

**Reply by ann0see** [reply]
That's still strange but glad it's working now.

**Reply by harmonk** [reply]
Yes, good news. On my side, i think i can report that Jamulus is working again.
I finally could make an external HD to boot from. I left Jamulus running a few times, and it never crashed. I had already reinstalled my OS on the computer. So, looks like a software clash of some kind, right?
I think my issue is different from bigzedd, as i never tried the local server thing, that i know.
What i did: i went through and deleted a bunch of old software, including lots of pesky old preferences from games etc. 
PageFlip.app
Scarlett MixControl.app
TechTool Pro 10.app
Tiny Player.app
Two Dots.app
Uninstall PlayOnMac.app
ZenTypist.app
-- are some. I am embarrassed to have to say that i did not keep a scientifically rigorous record. Sorry! 
I updated a few others, and deleted and reinstalled some, including Zoom.
And now Jamulus seems non-crashing. I did two tests of two hours so far. I will continue, and if it DOES crash you will be the first to know.
But anyway i want to thank you for the help and time and attention! Will keep you posted.

**Reply by ann0see** [reply]
Good to hear!

Thanks everyone also for helping out.
